config/env/development.js

```diff
@@ -1,7 +1,9 @@
 'use strict';
 
 module.exports = {
-	db: 'mongodb://localhost/mean-dev',
+	db: {
+		url: 'mongodb://localhost/mean-dev'
```

Comment:
please change it to `db: 'mongodb://localhost/mean-dev` to be consistent with 0.4.0.